pkg/ruler/base/storage.go

```diff
@@ -72,7 +70,7 @@ func (cfg *RuleStoreConfig) Validate() error {
 
 // IsDefaults returns true if the storage options have not been set
 func (cfg *RuleStoreConfig) IsDefaults() bool {
-	return cfg.Type == "configdb" && cfg.ConfigDB.ConfigsAPIURL.URL == nil
+	return cfg.Type == ""
```

Comment:
should we maybe default to local storage instead of nothing?

Reply:
I think the problem with defaulting to local storage is that it's not likely to work without configuration.
We can't assume that certain directories (or indeed even the filesystem) is writeable, and I'd rather push folks to have to configure the storage themselves. If we default local storage and they get errors, it'll be a bit confusing because they didn't explicitly set anything.